examples/read.py

```diff
@@ -2,20 +2,21 @@
 from os import uname
 
 
-def do_read():
+def do_read(x,y):
```

Comment:
pep8: please have a blank after the comma

also, don't call it x and y, but give good names.